grafana/dashboards/k6-prometheus.json

```diff
@@ -2014,13 +2042,13 @@
     ]
   },
   "time": {
-    "from": "now-5m",
+    "from": "now-6h",
     "to": "now"
   },
   "timepicker": {},
   "timezone": "",
   "title": "k6 Prometheus",
   "uid": "ccbb2351-2ae2-462f-ae0e-f2c893ad1028",
-  "version": 1,
+  "version": 6,
```

Comment:
@Sea-you, I don't think we need to change this, but nvm, can we set this to version `2` instead?

Reply:
fixed